systemtests/src/test/java/io/enmasse/systemtest/scale/ScaleTest.java

```diff
@@ -0,0 +1,224 @@
+import io.enmasse.admin.model.v1.AddressSpacePlan;
+import io.enmasse.admin.model.v1.AuthenticationService;
+import io.enmasse.admin.model.v1.ResourceAllowance;
+import io.enmasse.admin.model.v1.ResourceRequest;
+import io.enmasse.admin.model.v1.StandardInfraConfig;
+import io.enmasse.admin.model.v1.StandardInfraConfigBuilder;
+import io.enmasse.admin.model.v1.StandardInfraConfigSpecAdminBuilder;
+import io.enmasse.admin.model.v1.StandardInfraConfigSpecBrokerBuilder;
+import io.enmasse.admin.model.v1.StandardInfraConfigSpecRouterBuilder;
+import io.enmasse.systemtest.UserCredentials;
+import io.enmasse.systemtest.bases.TestBase;
+import io.enmasse.systemtest.bases.isolated.ITestBaseIsolated;
+import io.enmasse.systemtest.logs.CustomLogger;
+import io.enmasse.systemtest.logs.GlobalLogCollector;
+import io.enmasse.systemtest.model.address.AddressType;
+import io.enmasse.systemtest.model.addressspace.AddressSpaceType;
+import io.enmasse.systemtest.utils.AddressUtils;
+import io.enmasse.systemtest.utils.AuthServiceUtils;
+import io.enmasse.systemtest.utils.PlanUtils;
+import io.enmasse.systemtest.utils.TestUtils;
+import io.fabric8.kubernetes.api.model.PodTemplateSpecBuilder;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.slf4j.Logger;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.IntStream;
+
+import static io.enmasse.systemtest.TestTag.SCALE;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.greaterThan;
+
+@Tag(SCALE)
+class ScaleTest extends TestBase implements ITestBaseIsolated {
+    private final static Logger log = CustomLogger.getLogger();
+    private final String namespace = "scale-test-namespace";
+    final String addressSpacePlanName = "test-addressspace-plan";
+    final String queuePlanName = "test-queue-plan";
+    final String anycastPlanName = "test-anycast-plan";
+    final String authServiceName = "scale-authservice";
+    AddressSpace addressSpace = null;
+    final UserCredentials userCredentials = new UserCredentials("scale-test-user", "password$");
+
+    @BeforeEach
+    void init() throws Exception {
+        kubernetes.createNamespace(namespace);
+        setupEnv();
+    }
+
+    @AfterEach
+    void tearDown(ExtensionContext extensionContext) throws Exception {
+        GlobalLogCollector.saveInfraState(TestUtils.getScaleTestLogsPath(extensionContext));
+        kubernetes.deleteNamespace(namespace);
+    }
+
+    @Test
+    void testNumberOfSupportedAddresses() throws Exception {
+        int operableAddresses = 0;
+        int iterator = 0;
+        int failureThreshold = 12000;
+
+        while (true) {
+            log.info("Total operable addresses: {}", operableAddresses);
```

Comment:
This seems like a very verbose output that is not that useful unless test is failing.